core/src/main/java/com/yandex/yoctodb/v1/V1DatabaseFormat.java

```diff
@@ -29,8 +29,6 @@
  */
 @ThreadSafe
 public final class V1DatabaseFormat extends DatabaseFormat {
-    public final static int FORMAT = 7;
```

Comment:
Why is that removed?

Reply:
Because we moved to features description in previous pull request - but forgotten to good ol' FORMAT that left unused.